src/drivers/uart/src/debug_port.rs

```diff
@@ -29,7 +29,10 @@ impl<'a> Driver for DebugPort<'a> {
         for (_i, &c) in data.iter().enumerate() {
             let mut s = [0u8; 1];
             s[0] = c;
-            self.timer.sleep(4_000_000); // that's in fs
+            // 1 microsecond
+            for _j in 1..125 {
```

Comment:
_j should be 0..125, otherwise the length is 124.

Also, isn't it 0.96 us as it is now?

Reply:
Oh well, I will start at `0`, right. :) And yes, that makes it half a microsecond. Whoops.

I'll drop a comment regarding 32-bit support.